public/popup.html

```diff
@@ -9,6 +9,8 @@
   </head>
 
   <body class="no-transitions">
+    <div class="d-flex justify-content-end version">v<span class="string" id="version-string">0.0.0</span></div>
```

Comment:
I think the class `string` is no longer needed, now that you removed some styling.